lib/painted_rabbit/base.rb

```diff
@@ -40,7 +36,7 @@ def self.render(object, view: :default)
     # This is the magic method that converts complex objects into a simple hash
     # ready for JSON conversion
     def self.prepare(object, view:)
-      unless views.keys.include? view
+      unless views.has_view? view
```

Comment:
Right here, `views` is actually a `ViewCollection`. Also `view` is not a `View` object, but a view name.
Can we rename the variables here and throughout other places in this file?
So that we have `view_collection` for `ViewCollection` and `view_name` for view names.

Reply:
Good call! I started doing this (a tiny bit) changing instances of `view` to `view_name` but did not go far enough to change it everywhere. I think this will really help clear things up